src/TLuaInterpreter.cpp

```diff
@@ -12211,8 +12211,9 @@ std::pair<bool, bool> TLuaInterpreter::callLuaFunctionReturnBool(void* pT)
         }
         lua_pop(L, lua_gettop(L));
         if (error == 0) {
-            return {true, returnValue};
+            return {true, returnValue };
```

Comment:
This is odd in style guide checks :) Either add spaces on both sides or delete on both I guess.

Reply:
Yep fixed